CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# dbt_klaviyo_source v0.7.0
+[PR #18](https://github.com/fivetran/dbt_klaviyo_source/pull/18/) includes updates regarding the [September 2023](https://fivetran.com/docs/applications/klaviyo/changelog#september2023) changes to the Klaviyo connector.
+
+## 🚨 Breaking Changes 🚨:
+
+- We have removed these columns from the following tables:
+
+| **Table**                          | **Column**                                                                                                                                                                                                                             |
+|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CAMPAIGN | is_segmented
+| FLOW | customer_filter
+| FLOW | trigger
+
+- We have added these columns from the following tables:
+
+| **Table**                          | **Column**                                                                                                                                                                                                                             |
+|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CAMPAIGN | archived
+| CAMPAIGN | scheduled
+| FLOW | archived
+| FLOW | trigger_type
+| PERSON | last_event_date
```

Comment:
i feel like we should have a third column that shows what we rename these fields to (if we rename them) in the respective staging models

Reply:
also could be good to add DAG links to the affected source tables so people can quickly see the descriptions for these new fields

Reply:
same for the removed fields

Reply:
awesome, updated!